test/services/ReviewServiceSpec.scala

```diff
@@ -128,32 +129,107 @@ class ReviewServiceSpec extends UnitSpec with MockFactory with ReviewData with A
   }
 
   "Calling the changeStatus method" when {
-    "the ID identifies a valid process" should {
-      "return to indicate the process status was updated in the database" in new Test {
+    "the ID identifies a valid process" when {
+      "the status is submitted for 2i review" should {
```

Comment:
Change from Submitted for 2i review to submitted for fact check

Reply:
test data changed so updated description